README.md

```diff
@@ -121,6 +138,23 @@ let package = Package(
   ]
 )
 ```
+ANd then build your app or example app:
+
+```bash
+swift build
+```
+...and if updating from an older version, you might need to re-build yourapp, [eg MyApp] , with :
+
+```bash
+    swift package update 
+    swift package purge-cache 
+    swift package reset 
+    swift package clean 
+    rm -rf ~/.cache/org.swift.foundation.URLCache/
+    rm -rf ~/.cache/org.swift.swiftpm/
+    rm -rf .build/
+    swift build 
+```
```

Comment:
This is a great piece for troubleshooting. I'd say it deserves a new troubleshooting section and another \<details\> spoiler since not everybody will require it

Reply:
moved to its own TroubleShooting section , near the bottom, as I dont think it deserves its own TroubleShooting.md just yet.